plugins/_tests_/no-missing-parenthesis.test.js

```diff
@@ -0,0 +1,25 @@
+const stylelint = require("stylelint")
```

Comment:
These tests are great! I'm wondering if we can make the new stylelint plugins packages rather than individual files so we're not cluttering up the root package with dependencies only for these items. Can you look at https://github.com/castastrophe/postcss-splitinator for an idea of what to include? I like ava for lightweight testing and it plugs in nicely with c8 for code coverage reporting.

Reply:
Got it. I checked out your amazing repo and updated the codebase by adding the plugins as packages and added ava as the testing tool. Added yarn lint:tests to trigger the stylelint plugins tests with lerna. 
Let me know how this looks now. Thanks @castastrophe